purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -660,6 +659,11 @@ internal class PurchasesOrchestrator constructor(
 
     // region Private Methods
 
+    private fun shouldRefreshCustomerInfo(firstTimeInForeground: Boolean): Boolean {
+        return (firstTimeInForeground || deviceCache.isCustomerInfoCacheStale(appUserID, appInBackground = false)) &&
+            !appConfig.customEntitlementComputation
```

Comment:
Nit: but since `!appConfig.customEntitlementComputation` is simpler to compute, it would save checking whether the cache is stale if you had it first.

Reply:
Done in https://github.com/RevenueCat/purchases-android/commit/f24191703e7c626b22cc345b1362fab5886bc0ff